pkg/utils/miscellaneous_test.go

```diff
@@ -30,12 +30,6 @@ import (
 
 var _ = Describe("utils", func() {
 	Describe("#MergeStringMaps", func() {
-		It("should return nil", func() {
-			result := MergeStringMaps(nil, nil)
-
-			Expect(result).To(BeNil())
-		})
```

Comment:
Why was this test removed?

Reply:
While using a function that takes generic object input we can't pass nil.

Reply:
```
                It("should return nil", func() {
			var nilMap map[string]string
			result := MergeStringMaps(nilMap, nilMap)

			Expect(result).To(BeNil())
		})
```
This test case is already covered.

Reply:
> ```
>                 It("should return nil", func() {
> 			var nilMap map[string]string
> 			result := MergeStringMaps(nilMap, nilMap)
> 
> 			Expect(result).To(BeNil())
> 		})
> ```
> 
> This test case is already covered.

Do you mean `It("should return an empty map"...`? The `emptyMap := map[string]string{}` will be a non-nil map. The map in the case suggested is a nil map.